.conan/conanfile-pinned.txt

```diff
@@ -0,0 +1,20 @@
+[generators]
+cmake
+pkg_config
+
+[requires]
+pkg-config_installer/0.29.2@bincrafters/stable
+boost/1.71.0@conan/stable
+OpenSSL/latest_1.1.1x@conan/stable
+libcurl/7.66.0@bincrafters/stable
+libusb/1.0.23@bincrafters/stable
+gettext/0.20.1@bincrafters/stable
+gmp/6.1.2@bincrafters/stable
+zlib/1.2.11@conan/stable
+
+[options]
+libusb*:enable_udev=False
+gettext:shared=True
```

Comment:
Doesn't this add a new dependency on Linux? (`gettext()` is just in libc for Linux) Is it possible to only enable a dep on macos vs linux?

Reply:
@ericiles Any ideas?

Reply:
We could define separate conanfiles for Mac vs. Linux and have that logic in the CMake level. Let's discuss this more in tomorrow's meeting.

Reply:
Gettext removed in #8118. Commit 0f2f92c removes this dependency from the conanfiles.